modules/decompressinput.sh

```diff
@@ -28,7 +28,7 @@ function compression_type () {
   fi
   local -r disk_image="${1}"
   case "${disk_image}" in
-    *.img | *.sdimg)
+    *.img | *.sdimg | *.ext4 | *.wic | *.rpi-sdimg)
```

Comment:
Never seen the `rpi-sdimg` before. What's the difference there?

Reply:
`wic` and `rpi-sdimg` are image types. Just suffix that that is different, but they are disk images.

But `ext4` stands out to me. This is not an disk image, and probably should not be used for "disk image" input.

Reply:
I thought we could function on an ext4 image.  Is that not correct?

Reply:
No, we currently do not have support for that.

Reply:
OK.  I removed ext4.